hawc/apps/riskofbias/serializers.py

```diff
@@ -104,7 +104,7 @@ class Meta:
 
 
 class RiskOfBiasScoreSerializer(serializers.ModelSerializer):
-    overridden_objects = RiskOfBiasScoreOverrideObjectSerializer(many=True)
+    overridden_objects = RiskOfBiasScoreOverrideObjectSerializer(many=True, read_only=True)
```

Comment:
The override serializer is not really setup for writes, so we just ignore it on writes.